tests/test_subtemplates.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+from datetime import datetime
+import gnupg
```

Comment:
It looks like a number of these imports aren't being used.
Can you check and remove any others?